_i18n/ca.yml

```diff
@@ -8,10 +8,15 @@ menu:
     description_1: "Coopdevs és una associació de persones que té com a objectiu <strong>impulsar solucions tecnològiques per fomentar i promoure l’economia social</strong>. Tots els projectes en què participem són de codi obert i llicència lliure."
-    description_2: "Si voleu col·laborar amb nosaltres ens podeu trobar al fòrum <a href='http://community.coopdevs.org'>community.coopdevs.org</a> i a les plataformes <a href='https://github.com/coopdevs'>github.com</a> i <a href='https://gitlab.com/coopdevs'>gitlab.com</a>. També ens podeu escriure a <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>info@coopdevs.org</a> o visitar-nos al <a href='https://passatgenogues.wixsite.com/cwnogues'>Coworking Nogués</a> de Gràcia."
-    projects_title: "Projectes"
-    katuma_description: "<a href='http://www.katuma.org'>Katuma</a> és una plataforma cooperativa que facilita la creació i gestió de grups de consum. Es tracta d'una alternativa més justa i sostenible que l'oferida per les grans distribuïdores i els intermediaris comercials."
-    timeoverflow_description: "<a href='http://timeoverflow.org'>Time Overflow</a> és una plataforma de gestió de Bancs de Temps, un sistema d'oferta de serveis a canvi de temps. La plataforma facilita les tasques de gestió i intercanvi de serveis entre els membres d'un banc."
+    description_2: "Si vols col·laborar amb nosaltres ens pots trobar al fòrum <a href='http://community.coopdevs.org'>community.coopdevs.org</a> i a les plataformes <a href='https://github.com/coopdevs'>github.com</a> i <a href='https://gitlab.com/coopdevs'>gitlab.com</a>."
+    description_3: "Si necessites els nostres serveis escriu-nos a <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>info@coopdevs.org</a> i passa a visitar-nos al <a href='https://passatgenogues.wixsite.com/cwnogues'>Coworking Nogués</a> de Gràcia."
+    action_lines: "Linies d'actuació"
+    open_source: "Dinamització de comunitats Open Source"
+    opensource_description: "Som “open by default” Desenvolupar software lliure amb una comunitat activa al voltant és la nostra passió, tenim experiència en la creació i dinamitació de comunitats Open Source. <a href='https://community.coopdevs.org/c/floss-communities'>Saber-ne més</a>"
+    development: "Desenvolupament de software per l'ESS"
+    development_description: "Desenvolupem software lliure per projectes de l'Economia Social i Solidària. Necessites ajuda? <a href='mailto:info@coopdevs.org?subject=Hola%20Coopdevs%21&body=Estava%20mirant%20la%20vostra%20web%20i%E2%80%A6'>Contacta amb nosaltres</a>"
+    processes: "Optimització de processos empresarials"
+    processes_description: "Analitzem i optimitzem processos empresarials per a cooperatives, fomentem la intercooperació en els processos que impliquen IT. <a href='https://community.coopdevs.org/c/it-processes-for-coops'>Saber-ne més</a>"
```

Comment:
what about replacing IT with "tecnologia" or "tecnologies de la informació"? Might easier to understand for non-techies.

Reply:
"_les TIC_" is also a common term